package com.bridgelabz.annotation.exercise.exercise2;
import java.lang.System.*;

public class LegacyAPI {

    @Deprecated
    public void oldFeature(){
        System.out.println("Old Method");
    }

    public void newFeature(){
        System.out.println("New Method");
    }

}
